### PR TITLE
round rounds are rounder than rough edges

### PR DESCRIPTION
**Pull request by j0code**

### CODE REVIEW

**Comment by j0code** [comment]
> is there sth that still has to be done on this?

suppose you don't wanna branch f-rounds-cmd off this branch, I don't think so

**Comment by cfpwastaken** [comment]
?

**Comment by cfpwastaken** [comment]
image

**Review by cfpwastaken — APPROVED** [review]
approved

**Comment by j0code** [comment]
image
I did some more changes :)

File: lyrics.js [hunk 1]
```diff
@@ -28,6 +29,39 @@ export default class LyricsMan {
...
+					description: "There is no game running in this channel",
+					color: [230, 0, 0],
+					ephemeral: true
+				}]})
+				return
+			}
+
+			if(i.customId == "join_game") {
+				if(game.participants[i.user.id]) {
+					i.reply({ content: "You alr joined. The Game will start soon.", ephemeral: true })
```
Comment (cfpwastaken) [inline lyrics.js:50]: make this `already` pls

**Review by cfpwastaken — CHANGES_REQUESTED** [review]

**Review by cfpwastaken — APPROVED** [review]
Yes.

**Comment by cfpwastaken** [comment]
You can merge it, ill do the command stuff later.

**Pull request merged**